### DISCUSSION TITLE
Link to internal files: page not found

### DISCUSSION THREAD

**Discussion by lorenzoFabbri** [daattali/beautiful-jekyll#829] [Q&A]
I am building my website (`beautiful-jekyll` published with GitHub Pages) and I would like to do the following: inside one of the main pages (a `.md` file) I would like to insert a link to another `.md` file, so that I can organize my projects nicely.

The current structure of the directory is the following:

 - `_research_projects/project1.md`
 - `_congif.yml`
 - `all_projects.md`

In the file `all_projects.md` I want to insert a link to `_research_projects/project1.md`. Therefore I inserted in the first file the following:

    [Project 1](/_research_projects/project1.md)

The file `project1.md` contains just the following at the moment:

    ---
    layout: default
    ---

    This is a test!

In the file `_config.yml` I added the following:

    gems:
      - jekyll-relative-links
    
    relative_links:
      enabled: true
      collections: true

But it still does not work: while in the GitHub repository the link works (it goes to the file `_research_projects/project1.md` if I click on the link), if I click on the link from the `.github.io` webpage, I get the error message:

    Whoops, this page doesn't exist.
    Move along. (404 error)

**Comment by daattali** [comment]
Could you try your method and post here a link to the resulting page, so we can see what happens
